src/java/org/apache/cassandra/utils/SyncUtil.java

```diff
@@ -39,60 +37,20 @@
 import org.apache.cassandra.io.util.File;
 
 /*
- * A wrapper around various mechanisms for syncing files that makes it possible it intercept
+ * A wrapper around various mechanisms for syncing files that makes it possible to intercept
  * and skip syncing. Useful for unit tests in certain environments where syncs can have outliers
- * bad enough to causes tests to run 10s of seconds longer.
+ * bad enough to cause tests to run 10s of seconds longer.
  */
 public class SyncUtil
 {
     public static final boolean SKIP_SYNC;
-
-    private static final Field mbbFDField;
-    private static final Field fdClosedField;
-    private static final Field fdUseCountField;
-
     private static final Logger logger = LoggerFactory.getLogger(SyncUtil.class);
 
     static
     {
-        Field mbbFDFieldTemp = null;
-        try
-        {
-            mbbFDFieldTemp = MappedByteBuffer.class.getDeclaredField("fd");
-            mbbFDFieldTemp.setAccessible(true);
-        }
-        catch (NoSuchFieldException e)
-        {
-        }
-        mbbFDField = mbbFDFieldTemp;
-
-        //Java 8
-        Field fdClosedFieldTemp = null;
-        try
-        {
-            fdClosedFieldTemp = FileDescriptor.class.getDeclaredField("closed");
-            fdClosedFieldTemp.setAccessible(true);
-        }
-        catch (NoSuchFieldException e)
-        {
-        }
-        fdClosedField = fdClosedFieldTemp;
-
-        //Java 7
-        Field fdUseCountTemp = null;
-        try
-        {
-            fdUseCountTemp = FileDescriptor.class.getDeclaredField("useCount");
-            fdUseCountTemp.setAccessible(true);
-        }
-        catch (NoSuchFieldException e)
-        {
-        }
-        fdUseCountField = fdUseCountTemp;
-
         //If skipping syncing is requested by any means then skip them.
         boolean skipSyncProperty = Boolean.getBoolean(Config.PROPERTY_PREFIX + "skip_sync");
-        boolean skipSyncEnv = Boolean.valueOf(System.getenv().getOrDefault("CASSANDRA_SKIP_SYNC", "false"));
```

Comment:
this is a nit and you can considered it or not - we have `CassandraRelevantEnv` class where we should add `CASSANDRA_SKIP_SYNC`